
An exception error occured
The 'try except' is finished
